leaflet.photon.js

```diff
@@ -114,7 +114,7 @@ L.PhotonBaseSearch = L.PhotonBase.extend({
     },
 
     createResultsContainer: function () {
-        this.resultsContainer = L.DomUtil.create('ul', 'photon-autocomplete', document.querySelector('body'));
+        this.resultsContainer = this.options.resultsContainer || L.DomUtil.create('ul', 'photon-autocomplete', document.querySelector('body'));
```

Comment:
As discussed: Changed the `createResultsContainer()` so that the user can change the container- and the parent-element via the `resultsContainer` option.